update JSON Forms to 3.2.0-beta.0

LukasBoll opened the pull request.

LukasBoll commented:
Hi @sdirix,
I updated the angular version, which needs a newer node version.
We need to update the node version to >18.13 in our build step to build the project.

lucas-koehler commented on package.json:
```diff
@@ -12,38 +12,38 @@
+    "@angular/common": "^17.0.8",
+    "@angular/compiler": "^17.0.8",
+    "@angular/core": "^17.0.8",
+    "@angular/flex-layout": "^15.0.0-beta.42",
```
`@angular/flex-layout` should not be needed anymore. Thus the dependency can be removed

lucas-koehler reviewed (changes requested):
Hi @LukasBoll, I did not test this yet but I noticed you used different Angular versions for the dependencies. Some use 16.x versions and some 17.x versions. All of them should use Angular 16 (i.e. the lower supported version).

LukasBoll commented:
> Hi @LukasBoll, I did not test this yet but I noticed you used different Angular versions for the dependencies. Some use 16.x versions and some 17.x versions. All of them should use Angular 16 (i.e. the lower supported version).

Hi @lucas-koehler, 
Thanks! I changed the angular version consistent to 16 and removed the unneeded flex-layout

lucas-koehler reviewed (approved):
Hi @LukasBoll thanks for the update! I added an `.nvmrc` file to specify the node version used by Netlify and updated the dependencies and unit testing a bit.
